### ISSUE TITLE
no sounds after convert

### ISSUE THREAD

**Issue by creativeart** [paator/eJayDecompressor#2]
Hello,
I tried the PXDConverter with Dance Ejay 2 (Millenium Edition) with the files which are in "PXD" folder. I copy them on my harddrive. Then I uyse the PXDConverter, it creates only 3-4 wav files as you can see here almost blank sound as you can see on this video :
https://drive.google.com/file/d/1gz02jMDinf7PenFwS7S0BtMlEfvb4H1n/view

Am I missed something ? I know that Dance Ejat 2 contains thousand of sounds.

Thank you so much.

**Comment by paator** [comment]
Hey, you don't need to copy pxd files.
Run PXDConverter.exe -all in your ejay installation folder. It will automatically search for all .pxd files inside all folders recursively. Perhaps there are some .pxd files that you have missed - converter will find these files:)

Tell me if it works!
Cheers

**Comment by creativeart** [comment]
Hello, thank you for your answer.
Since I run Windows 10, I cannot install Dance ejay 2 anymore on this pc because it's not compatible with Dance Ejay 2. That's why I copy/paste the PXD file from the CD. Must I copy/paste all the CD on my hard-drive ? Not only the PXD so ?

**Comment by paator** [comment]
Hmmmm perhaps some of their .pxd files are unpacked only after installation. If these 3 files are the only ones you could find with .pxd extension, my best bet is that probably more of them would show up after installation. Try to copy-paste all files from CD into one folder on hard drive and run my script there. If it doesn't find more files, you would need to try installing Dance ejay 2 on older machine - or virtual machine (for example Virtual Box) with older operating system installed to see if more .pxd files would show up. Can you also provide me a screenshot of files on your ejay CD? Maybe there is an archive containing all pxd files which is unpacked upon installation? If that's the case I may try to unpack these files in my converter.

Cheers

**Comment by creativeart** [comment]
Ok I will try to find an older machine. Thank you for the advise. In attached file, I put some screenshots from cd ejay CD. The files I tried was the big files from PXD folder.
![Capture](https://user-images.githubusercontent.com/4004589/119452069-392bfd80-bd36-11eb-8d50-bb690e3592de.JPG)
![Capture2](https://user-images.githubusercontent.com/4004589/119452071-39c49400-bd36-11eb-91c4-35c23d50284e.JPG)
![capture3](https://user-images.githubusercontent.com/4004589/119452073-39c49400-bd36-11eb-91ee-ea56eaa5a761.JPG)

**Comment by paator** [comment]
These PXD samples may be used by installer itself. Some installers back then had weird audio effects..:)
Im sure that more PXD files will show up after installation.

**Comment by creativeart** [comment]
Perfect, thank you, I will try on an older computer with Windows XP later. Thank you for your help :) I will tell you if it works.

**Comment by creativeart** [comment]
Ok it works for Dance Ejay 1 but I don't find any PXD for Dance Ejay 2 Millenium edition. Here is a picture I took after installing Dance ejay 2 on my Windows XP computer :/ Any ideas ? 
![IMG_20210525_201442](https://user-images.githubusercontent.com/4004589/119973918-58d05980-bfb4-11eb-9d64-bcc1f70f3671.jpg)

Anyway thank you so much for this tool. I'm happy to listen to my samples from Dance Ejay 1 ;)

I tried Dance Ejay 3 and Dance Ejay 4. But it's the same than Dance Ejay 2, It's only some 2-3 PXD files :( It's not a PXD per sounds. Any ideas to be able to extract these ones ? I can send you the file or ISO if it can help :)
